pkg/commands/docker.go

```diff
@@ -274,6 +275,22 @@ func (c *DockerCommand) filterOutExited(containers []*Container) []*Container {
 	return toReturn
 }
 
+// sortedContainers returns containers sorted by state if c.SortContainersByState is true (follows 1- running, 2- exited, 3- created)
+// and sorted by name if c.SortContainersByState is false
+func (c *DockerCommand) sortedContainers(containers []*Container) []*Container {
+	if !c.Config.UserConfig.Gui.SortContainersByName {
+		sort.Slice(containers, func(i, j int) bool {
+			states := map[string]int{
```

Comment:
Given that this is effectively just a constant, we should pull it out of the loop to spare ourselves the unnecessary memory allocations

Reply:
Done 😄